src/components/buttons/hold-to-authorize/HoldToAuthorizeButtonContent.tsx

```diff
@@ -228,7 +235,7 @@ function HoldToAuthorizeButtonContent2({
 
   let buttonLabel = label;
   if (isAuthorizing) {
```

Comment:
This is a nit, and I am not sure it will work as it should, but it could be less nested and simpler if you separated the nested if into two if statements like
```
if (isAuthorizing && isHardwareWallet) {
	buttonLabel = lang.t(
		lang.l.button.hold_to_authorize.confirming_on_ledger
	);
}
if (isAuthorizing) {
	buttonLabel = lang.t(lang.l.button.hold_to_authorize.authorizing);
}
```